velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,28 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, cauchyCDF) {
+  const auto cauchyCDF = [&](std::optional<double> median,
+                             std::optional<double> scale,
+                             std::optional<double> value) {
+    return evaluateOnce<double>("cauchy_cdf(c0, c1, c2)", median, scale, value);
+  };
+
+  EXPECT_EQ(0.5, cauchyCDF(0.0, 1.0, 0.0));
+  EXPECT_EQ(0.75, cauchyCDF(0.0, 1.0, 1.0));
+  EXPECT_EQ(0.25, cauchyCDF(5.0, 2.0, 3.0));
+  EXPECT_EQ(1.0, cauchyCDF(5.0, 2.0, kInf));
+  EXPECT_EQ(0.5, cauchyCDF(5.0, kInf, 3.0));
+  EXPECT_EQ(0.0, cauchyCDF(kInf, 2.0, 3.0));
+  EXPECT_EQ(1.0, cauchyCDF(5.0, 2.0, kDoubleMax));
```

Comment:
Any reason you didn't write a test where kDoubleMax is the first parameter ?

Reply:
@aditi-pandit No reason. Added in latest commit.